src/lib/graphql_lib/base_types.ml

```diff
@@ -23,3 +23,7 @@ let uint32 () =
 
 let uint64 () =
   unsigned_scalar_scalar ~to_string:Unsigned.UInt64.to_string "UInt64"
+
+let token_id () =
+  scalar "TokenID" ~doc:"String representation of a token's UInt64 identifier"
```

Comment:
nit: Unify capitalization between `TokenID` here and `TokenId` above (I prefer the lowercase "d" at the end)